### PR TITLE
feat(tooltip): make popper options versatile

### PR DESCRIPTION
**Pull request by larsdouweschuitema**
Make the tooltip (popper) options versatile instead of offering strict placement support.

Also deleted the key because I don't think it offers any value.

### CODE REVIEW

**Review by Zishrodrigues — APPROVED** [review]

File: src/stories/ui-tooltip.stories.js [hunk 1]
```diff
@@ -41,8 +40,15 @@ const Template = (args, { argTypes }) => ({
 export const Tooltip = Template.bind({});
 Tooltip.args = {
     background: 'light',
-    trigger: 'hover',
-    side: 'bottom',
+    trigger: 'click',
+    options: {
```
Comment (dine) [inline src/stories/ui-tooltip.stories.js:44]: Maybe we can add the link to the vue-popperjs docs somewhere in this stories file? https://github.com/RobinCK/vue-popper#props

File: src/stories/ui-tooltip.stories.js [hunk 2]
```diff
@@ -28,7 +27,7 @@ export default {
 const Template = (args, { argTypes }) => ({
     props: Object.keys(argTypes),
     components: { UiTooltip },
-    template: `<ui-tooltip :trigger="trigger" :side="side" :background="background">
+    template: `<ui-tooltip :trigger="trigger" :options="options" :background="background">
```
Comment (dine) [inline src/stories/ui-tooltip.stories.js:30]: This will be a breaking change

**Review by dine — CHANGES_REQUESTED** [review]
can you add the "BREAKING CHANGE" message to the commit body?

**Review by dine — CHANGES_REQUESTED** [review]
can you add the "BREAKING CHANGE" message to the commit body?

File: src/stories/ui-tooltip.stories.js (hunk 2 above)
Comment (larsdouweschuitema) [reply src/stories/ui-tooltip.stories.js:30]: - [ ] make breaking change

File: src/stories/ui-tooltip.stories.js (hunk 1 above)
Comment (larsdouweschuitema) [reply src/stories/ui-tooltip.stories.js:44]: - [ ] add link to doc

**Review by dine — APPROVED** [review]

**Pull request merged**